apps/communications/pbap/js/pbap.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+/*
+    Phone Book Access Profile (PBAP) is a profile that enables devcices to
```

Comment:
nit: Please keep the indentation of comments consistent and use jsdoc comments, for example here I'd prefer
```
/**
 * Phone Book Access Profile (PBAP) is a profile that enables devices to
 * exchange phonebook objects via Bluetooth. After a Bluetooth connection is
 * established, ...
 */
```

Likewise for the other comments.